cMake/FreeCAD_Helpers/InitializeFreeCADBuildOptions.cmake

```diff
@@ -1,200 +1,236 @@
-macro(InitializeFreeCADBuildOptions)
+macro (InitializeFreeCADBuildOptions)
     # ==============================================================================
-    # =================   All the options for the build process    =================
+    # =================   All the options for the build process =================
     # ==============================================================================
 
-    option(BUILD_FORCE_DIRECTORY "The build directory must be different to the source directory." OFF)
-    option(BUILD_GUI "Build FreeCAD Gui. Otherwise you have only the command line and the Python import module." ON)
-    option(FREECAD_USE_EXTERNAL_ZIPIOS "Use system installed zipios++ instead of the bundled." OFF)
-    option(FREECAD_USE_EXTERNAL_SMESH "Use system installed smesh instead of the bundled." OFF)
-    option(FREECAD_USE_EXTERNAL_KDL "Use system installed orocos-kdl instead of the bundled." OFF)
-    option(FREECAD_USE_EXTERNAL_FMT "Use system installed fmt library if available instead of fetching the source." ON)
-    option(FREECAD_USE_EXTERNAL_ONDSELSOLVER "Use system installed OndselSolver instead of git submodule." OFF)
-    option(FREECAD_USE_FREETYPE "Builds the features using FreeType libs" ON)
-    option(FREECAD_BUILD_DEBIAN "Prepare for a build of a Debian package" OFF)
-    option(BUILD_WITH_CONDA "Set ON if you build FreeCAD with conda" OFF)
-    option(BUILD_DYNAMIC_LINK_PYTHON "If OFF extension-modules do not link against python-libraries" ON)
-    option(INSTALL_TO_SITEPACKAGES "If ON the freecad root namespace (python) is installed into python's site-packages" ON)
-    option(OCCT_CMAKE_FALLBACK "disable usage of occt-config files" OFF)
+    option (BUILD_FORCE_DIRECTORY "The build directory must be different to the source directory."
```

Comment:
The inhomogenous splitting of this block looks quite messy. Would be good to use format off/on here even if some lines have more than 100 characters.

Reply:
OK, will do.